SolutionStack/sysdig/adoc/TRD-Kubernetes-ss-sysdig-gs.adoc

```diff
@@ -0,0 +1,247 @@
+:docinfo: shared
+include::./common_docinfo_vars.adoc[]
+
+// = {title}
+= Sysdig with SUSE Rancher: Getting Started Guide
+
+// SUSE Rancher - Sysdig
+// :author: Samip Parikh, Solutions Architect, SUSE
+// :author: Manuel Boira Cuevas, Strategic Partners Solutions Architect, Sysdig
+:revnumber: 0.0.1
+:toc2:
+:toc-title: Sysdig with SUSE Rancher: Getting Started Guide
+
+:toclevels: 4
+
+:productname: SUSE Rancher
+
+
+== Introduction
+
+=== Motivation
+
+SUSE Rancher enables you to accelerate your digital transformation by unifying your cloud native application landscape, ensuring consistent operations and workload management from core to cloud to edge.  Sysdig Secure gives you enhanced visibility and is designed to enable DevOps teams simplify security in cloud native environments, from development through production.
+
+Together, Sysdig Secure provides a number of capabilities to help you securely run your applications in your SUSE Rancher Kubernetes landscape:
+
+* *Secure the build* – Automate image scanning in CI/CD pipelines and registries, efficiently flag vulnerabilities and owners, and block risky images.
+* *Detect and respond to threats* – Identify threats across containers and cloud services and prevent intrusion and lateral movement. Respond quickly and conduct forensics using detailed activity audits.
+* *Continuously validate cloud configurations and compliance* – Identify misconfigurations and compliance violations, and measure progress with detailed reports. Save time with out-of-the-box policies for standards such as PCI, NIST, HIPAA, and SOC 2.
+* *Monitor and troubleshoot infrastructure and services*  – Maximize performance and availability using Kubernetes and cloud monitoring with full Prometheus compatibility.
+
+
+=== Scope
+
+This guide will help you take the first steps to integrate SUSE Rancher with the Sysdig Secure SaaS solution.
+
+
+=== Audience
+
+This document is intended for DevOps and DevSecOps engineers, security professionals, and developers who are responsible for enabling security across containers, clusters, and cloud.
```

Comment:
We were vague about "cloud," as it could be public clouds or private clouds.  Sysdig provides enhanced security over what is typically available in a public cloud as well as unification of security data across platforms.  We should probably state this more clearly.